### ISSUE TITLE
pspnet

### ISSUE THREAD

**Issue by Skype111** [divamgupta/image-segmentation-keras#116]
Segmentation results are so terrible to switch to pspnet，why?

**Comment by Adelost** [comment]
Wondering this myself. I also noticed that the output is 1/3 of the input size for pspnet instead of 1/2 for unet, is a larger input size required to get comparable accuracy to unet?

**Comment by divamgupta** [comment]
Are you training from scratch or finetuning the pre-trained model. Finetuning the pretrained model could yield better results.
